spock-specs/SpockTestConfig.groovy

```diff
@@ -1,5 +1,10 @@
+import org.spockframework.specs.extension.Snapshot
+
 runner {
   parallel {
     enabled true
   }
+  if (snapshots.updateSnapshots) {
+    include Snapshot
```

Comment:
This whole thing could be simpler as all the current users have the suffix `AstSpec`. You could also just introduce a common superclass.

Reply:
Of course, I thought about that.

My intention was, that you cannot easily miss it.
If you go on a common superclass, you could forget to extend it, if you go for `*AstSpec`, you could accidentally choose a different name either intentionally or e.g. by typo.
If it is automatically done by applying the annotation, you are just on the safe side.

But if you want to keep it as field annotation and just require the `AstSpec` naming convention for it to work properly, that's fine with me too. Just tell me how you want it and I change it accordingly.

Reply:
Actually using `*AstSpec` might be bad, I have another use-case in mind where I will use a snapshot test but it has nothing to do with testing the AST and I don't think it has a genric but specific enough category to use for another naming convention except maybe `...SnapshotSpec` or similar.

Btw. if the AST tests are the main use-case, why is the default extension txt? Should we also change that to `groovy`?